src/test/java/paymelah/testutil/DebtListBuilder.java

```diff
@@ -12,16 +10,12 @@
  * A utility class to help with building DebtList objects.
  */
 public class DebtListBuilder {
-    public static final Debt DEFAULT_DEBT = CHICKEN_RICE;
-
     private List<Debt> debtList = new ArrayList<>();
 
     /**
-     * Creates a {@code DebtListBuilder} with a default debt.
+     * Creates an empty {@code DebtListBuilder}.
      */
-    public DebtListBuilder() {
-        debtList.add(DEFAULT_DEBT);
-    }
+    public DebtListBuilder() {}
```

Comment:
The other builder classes have private constructors to prevent instantiation - maybe this should be private as well?

Reply:
What do you mean? You're meant to be able to instantiate builders, I also can't find a builder class that has a private constructor

Reply:
Never mind I got very confused at 2am xD Ignore this comment lol